pkg/vpcmodel/nodesConnectivity.go

```diff
@@ -21,7 +21,7 @@ import (
 // (2) compute AllowedConnsCombined (map[Node]map[Node]*connection.Set) : allowed conns considering both ingress and egress directions
 // (3) compute AllowedConnsCombinedStateful : stateful allowed connections, for which connection in reverse direction is also allowed
 // (4) if grouping required - compute grouping of connectivity results
-func (c *VPCConfig) GetVPCNetworkConnectivity(grouping bool) (res *VPCConnectivity, err error) {
+func (c *VPCConfig) GetVPCNetworkConnectivity(grouping, lbAbstraction bool) (res *VPCConnectivity, err error) {
```

Comment:
`lbAbstraction` is relevant only for `Drawio`, right?
Is there a way to avoid adding this parm to this function which is used by "everyone"?

Reply:
lbAbstraction  is not only for drawio.
it will be a user input, and be reflected in all output formats.